# Lecture 7: Proteins
Proteins have many functions and are fundamental aspects of the cell.

## The Building Blocks
For proteins, the monomers are the *amino acids* and the polymers are called * s*, *polypeptides* and *proteins*.  

## Structure of Amino Acid
The structure of an amino acid contains an amino, carboxyl and R groups. The aspect that distinguishes one amino acid from another is the *side chain* (the R group) which consist of hydrocarbons, oxygen's, carboxyl groups or other amino groups.

## Types of Amino Acids
There are 20 types of *amino acids*, which only differ in the R group. These groups are sometimes hydrophobic, hydrophilic or carry a charge at the cellular pH.

Hydrophobic amino acids have non polar side chains such as H or CH3.  Hydrophilic amino acids have polar side chains such as CH2-OH or CH2-SH. Some of the hydrophilic groups also have electrically charged side chains such as Aspartic Acid or Lysine.

## Bonds
The polymerization of amino acids include the formation of *peptide bonds*. This reaction involves the amino group on one amino acid and the carboxyl group on the other. Similar to carbohydrates, this reaction is also a dehydration reaction. This reaction occurs on the ribosome.

A *tripeptide* is three amino acids covalently bonded together. This molecule is divided has two sections named the *backbone*, everything except the R group, and *side chains*, the R groups.

## Structure of Proteins
There are four levels of protein structure.

1. *Primary Stucture*: The sequence of amino acids. There is an amino end and a carboxyl end.
2. *Secondary Structure*: The coils and folds between the polypeptide chains as a result of the H bonds between the backbone atoms. The α helices are flexible whereas the β sheets are rigid. As a result, proteins can form *alpha helix* or *pleated sheets*. Some toxic proteins may go through α helix to β sheet transition. These bonds are formed between the carbonyl O of one animo acid and the amino H of another another.
3. *Tertiary Structure*: The coils and folds between the polypeptide chains as a result of the interactions among the various R groups. The interactions stabilize the tertiary structure of the protein through the user of all kinds of bonds such as hydrogen/ionic bonds and hydrophobic/van der Waals interactions. One special example is the *disulfide bridge*, a covalent bond, which occurs between cysteine during oxidation reactions.
4. *Quaternary Structure*: When a protein consists of multiple polypeptide chains. Two or more polypeptide groups (subunits) may interact. For example, this may create fibrous proteins such as *collagen* or globular proteins such as *hemoglobin*. This structure is mostly held together with the same bonds as the tertiary structure.

## Functional Groups
Protein structure can be modified by the addition of functional groups. For example, a phosphate group can be added to the R group through the use of a covalent bond. This introduces a negatively charged functional group which changes the shape (and function) of the protein.

## How is Structure Determined?
Proteins structure is determined experimentally through methods such as *x-ray crystallography* and *nuclear magnetic resonance spectroscopy*. Tertiary structure may also be predicted from the primary structure.

The structure is influenced by factors such as pH, salt concentration and temperature.

*Denaturation* is the destruction of the weak bonds and interactions that maintain the 3D structure and leads to an inactive protein called *denatured proteins*. *Renaturation* is the opposite process.

# Lecture 8: Getting to Know Proteins Better
Each protein has a unique shape (tertiary structure) that determines function. The shape is determined by the primary structure and the side chains of amino acids determine the interactions that occur within proteins.

## Proteins & Disease
The misfolding of proteins leads to many neurodegenerative diseases such as *Alzheimer's disease*, *Huntington's disease*, *Parkinson's disease* and *prion diseases*. The risk factor increases with age due to the accumulation of misfolded proteins that aggregate. These proteins damage the nerves.

### Alzheimer's Disease
Alzheimer's is a degenerative brain disorder. An individual who has this disease has a progressive loss of abstract thinking and memory. During the onset, misfolded *amyloid* proteins (*β-amyloid* protein) begin to aggregate to form *amyloid plaques*.

*Amyloid-β precursor proteins* (*APP*) are transmembrane proteins in nerve cells which normally act as signal receptors involved in neural growth and repair.

The *cleavage* of APP molecules, cut by *proteases*, changes during the onset of the disease. This abnormal cleavage produces neurotoxic Aβ peptides (40-23 amino acids) which aggregate to form amyloid plaques.

Most cases of Alzheimer's are not inherited; however, some forms of the disease can be inherited. For example, the amino acid at position 22 of the peptide may be G instead of E which creates a peptide called *Aβ22G*.

# Lecture 9: Nucleic Acids
There are two types of *nucleic acids* including:
* *Deoxyribonucleic acid* (*DNA*)
* *Ribonucleic Acid* (*RNA*)

The functions of DNA including the storage of hereditary information and to act as protein blueprints. The functions of RNA include the transmission of instructions for making proteins, regulating the synthesis of proteins and helping directly in the synthesis of proteins. The flow of genetic information is DNA -> mRNA -> Protein with help of *Transfer RNA* (*tRNA*), *Ribosomal RNA* (*rRNA*) and *Regulatory RNA*.

## Structure
The monomers of nucleic acids include *nucleotides*. The nucleotide includes a varying amounts of phosphate groups, a pentose sugar and *nitrogenous base*. The sugar and base together are called the *nucleoside*. The full structure name will vary depending on the number of phosphate groups, such as *nucleoside monophosphate* and *nucleoside triphosphate*.

Within the sugars, DNA contains a *deoxyribose sugar* with two hydrogens on the 2' carbon whereas RNA contains a *ribose sugar* with on OH on the bottom. The phosphate group attaches to the 5' carbon on the sugar. The OH on the 3' carbon is used for the polymerization of nucleotides. This means the growing point is the 3' carbon and the *directionality* is 5'->3'.

Polymerization is completed through the use of dehydration reactions catalyzed by *DNA polymerase* to form *phosphodiester linkages*. During this reaction, the OH of the nucleoside triphosphate of one strand reacts with the OH of the 3' carbon. The OH on the phosphate group is completely removed and the other two phosphate groups break off to form *pyrophosphate*. The polymer of nucleotides are *nucleic acids*.

Nucleic acids have a certain sequence of bases which is how information is stored. This sequence is supported by the *sugar-phosphate backbone* and details the sequence of amino acids in proteins.

RNA is a *single stranded* whereas DNA is *double stranded* (has two antiparallel strands) which allows it to twist around itself to form a *double helix*. Within the double helix, there are *major grooves*, where the backbones from one level to the next are far apart, and *minor grooves*, where they are close together. To create double stranded nucleic acids, hydrogen bonds are used to hold them together. There exists two hydrogen bonds between the A & T and three hydrogen bonds between the C & G.

## Types of Nucleotides
There are five types of nucleotides which differ in their nitrogenous bases.
1. *Cytosine*
2. *Thymine*
3. *Uracil*
4. *Adenine*
5. *Guanine*

Uracil is present on RNA whereas thymine is present on DNA.

## Who Discovered the Structure
*Francis Crick* and *James Watson* were the two who discovered the structure; however,  Rosalind Franklin aided in the discovery. *Erwin Chargaff* created *Chargaff's Rule of Base Pairing* where %A = %T and %C = %G.

## Replication
The structure of DNA is important for easy replication and for the transmission of genetic material. The same sequence of nucleotides is passed on from parent to child. Each strand serves as a template for the construction of a complementary strand.

RNA also has a secondary structure due to *intrastrand base pairing* (it will pair with itself).

## DNA in Prokaryotic Cells
Prokaryotic cells usually have one large circular DNA molecule called a *chromosome*; however, is sometimes several smaller circular DNA molecules called *plasmids*. This DNA is not enclosed within a membrane and is *supercoiled* into a *nucleoid*.

## DNA in Eukaryotic cells
In eukaryotic cells, DNA is found within the nucleus bound to *histone proteins* via charged amino acids to form *chromatin*. DNA that is wound around a histone is called a *nucleosome* which look like beads on a string. The interactions between the nucleosomes create chromatin which coils into fibers. Further looping of the fibers condenses them into *chromosomes*. A *chromatid* is one strand of the chromosome pair. Some regions, called *heterochomatin*, are tightly packed and inactive. Other regions that are less compacted are called *euchromatin* which are active.

## Genes
*Genes* are discrete units of hereditary information. They are specific linear segments of DNA. Each gene occupies a distinct region on the chromosome.

# Lecture 10: DNA Replication
DNA replication is a key process for all living cells. It must be copied accurately and the process must be able to be carried out easily.

## Process
The high level process is:
1. Separation: DNA strands separate
2. Enzymes bind and use each strand as a template

It is a *semiconservative* process as each new double helix consists of one new and one old strand. It is also a fact process (50-500 nucleotides / second) and an accurate process (1 mistake / 10 billion nucleotides). The whole process required more than ten different proteins.

### Separation
The replication process begins with the seperation of the DNA strands. The site of seperation is an AT rich sequence at the site of replication. The origin is a specific sequence of nucleotides. An *initiator protein* binds to the DNA and uses ATP to slightly unwind the DNA. Within eukaryotic cells, replication begins at several site simultaneously.

*Helicase* then begins to fully unwinds these strands in both directions by breaking the hydrogen bonds. *Single-strand binding proteins* protect the unwound ssDNA and help hold the DNA strands apart during replication. While unwinding, the strands also become very twisted. Another protein called *topoismomerase* (*DNA Gyrase*) breaks, untwists and rejoins the strands ahead of the DNA helicase.

### Primers
After the seperation of the strands, an RNA primer needs to be attached to start the replication process. *Primers* are RNA sequences which bind to the specific sequence. These primers provide the initial 3' OH and are constructed by *primase*. Primase binds to the first initiating sequence to do this. The leading strand only needs one primer constructed; however, the lagging strands requires multiple primers.

## Synthesis
After separation begins, *DNA polymerase III* catalyze the dehydration reactions necessary for the addition of nucleotides to make the new strands of DNA. This reaction also requires a 3' OH group as the directionis always 5' -> 3'.

DNA polymerase only catalyzes the addtion of nucleoside triphosphates to the 3' end; however, since there is a replication fork the actual replication occurs in both directions simultaneously.

DNA polymerase also features a sliding clamp to reattach the hydrogen bonds. The synthesis of the *leading strand* is continuous whereas the synthesis of the *lagging strand* is discontinuous. As the lagging strand has multiple DNA fragments, called *okazaki fragments*, there are multiple RNA fragments. *DNA polymerase I* replaces the RNA primers with DNA. *DNA ligase* links the okazaki fragments.

*DNA polymerase* detects and corrects errors during replication. Later, other enzymes also do some "proofreading".

## Termination
In prokaryotes, replication continues until the whole circular strand is complete. In eukaryotes, chromosome replication terminates at *telomeres*. In humans, the sequence consists of TTAGGG repeated ~2500x.

Each replication cycle, the lagging strand is shortened by the primer length. Humans can lose up to 300 base pairs of *telomeric DNA* per replication. After 50-70 divisions, most cells die; however, some cells maintain their telomeres using *telomerase* which catalyzes the addition of *telomeric sequences*. This protein is produced in the vast majority of cancer cells.

# Lecture 11: Transcription
Most DNA encodes proteins (the rest encode RNA). Specific sequences, called *genes*, of DNA encode specific proteins. This process is similiar in both eukaryotes and prokaryotes.

This *transcription* process begins and ends at specific sequences of nucleotides. *mRNA* is produced from one DNA strand. During the *translation* phase, a polymer of amino acids called a *polypeptide* is created.

## Prokaryotes vs. Eukaryotes
Transcription and translation occur simultaneously in prokaryotes at no specific location.

In eukaryotes, the two processes are spatially separated by the nucleus. There may be additional processing that occurs before transport.

Multiple *ribosomes* may be attached to one mRNA molecule.

## Regulatory Sequence & Mechanisms of Transcription
The *regulatory sequence* is a region "upstream" of the beginning of the gene. It controls factors such as timing, location and amount of gene expression. The beginning of the transcription starts at a predetermined location called the *promoter*. This is the binding site for RNA polymerase.

The *transcription start site* is the first nucleotide in the mRNA that is produced which is located before the end of the promoter. The *transcription unit* is the full mRNA sequence that is copied. The *protein coding sequence* is only a subunit of this sequence.

A eukaryotic promoter includes a *TATA box*. Before *RNA polymerase II* can find, several *transcription factors* must bind to the DNA. After the RNA polymerase has bound, additional transcription factors are bound forming the *transcription initiation complex*.

RNA polymerase II separates by melting and catalyzes the polymerization of RNA 5' 3'.

For RNA, U pairs with A instead of T.

# Lecture 12: RNA Processing
Enzymes in the nucleus modify *pre-RNA* before it is transported out. During this event, both ends of the pre-RNA are altered. In addition, some interior parts are also cut out and spliced together.

## End Modification
At the 5' end a *cap* is added and at the 3' end *poly-A tail* is added. This allows RNA to be exported, protects it from degradation and help the ribosome bind.

The cap is *guanosine triphosphate*. It forms a triphosphate linkage. A methyl group is also added to the cap at N7 to form 7-methylguanosine.

A complex of proteins recognizes a *polyadenylation signal* in the *3'UTR*, the section of the mRNA that immediately follows the termination codon, and initiates the *polyadenylation* process. The complex cleaves the 3' end and adds a long chain of *adenine nucleotides* called the poly-A tail (long sequence of A's).

## Interior Modification
Within the mRNA, *introns* are intervening sequences that are transcribed but removed. *Exons* are sequences that remain and that are spliced together.

The process is completed by a large complex of proteins and RNA called a *spliceosome*. One of the key items is called *snRNPs* which stands for *small nuclear ribonucleic proteins* which consists of snRNA and protein.

The intron ends are recognized by the snRNAs by binding to specific sequences at the ends via complementary base pairing, like usual. There are different kinds of snRNAs which bind to other locations on the intron. The ends of the introns are then cut at these splice sites and the exons are joined together.

Introns exist for many reasons. Some introns control gene expression. Furthermore, some genes code for multiple polypeptides depending on which areas are treated as exons during splicing. This is called *alternative RNA splicing*. For example, the *doublesex* gene on the fruit fly *Drosophila melanogaster* produces two different sex proteins. About 60% of human genes are spliced into more than one different mRNA for a total of ~100,000 proteins. For example, the muscle protein *α-trpomysin* codes for the heart muscle and uterine muscle. 

# Lecture 13: Translation
During this process, *tRNA* carries amino acids and three base pairs. These base pairs match up with specific three base pair long mRNA sequences called *codons*.

## Transfer RNA
There are many types of *transfer RNAs* (tRNAs) in the cell. Each has a specific *anticodon* sequence and carries a specific amino acid. An anticodon matches with a complementary codon. The anticodon must be antiparallel so that the functional groups match up and the bonds are able to be formed.

*Aminoacyl-tRNA synthetase* match tRNA with a specific amino acid. There are 20 different kinds of this protein, all which catalyze the formation of a covalent bond between the tRNA and an amino acid to form *aminoacyl tRNA*. This process is called *charging tRNA*. The amino acid attaches to the 3' end of the tRNA.

## Ribosomes
The ribosome is divided into a large sub-unit and a small sub-unit which join over the mRNA. *Ribosomal RNAs* (*rRNAs*) catalyzes the formation of *peptide bonds* between the amino acids.

The large subunit contains three sites. The *peptidyl-tRNA bind site* (*P-site*) holds tRNA with attached peptide chain. The *aminoacyl-tRNA binding site* (*A-site*) holds tRNA holds the amino acid to be added. The *exit site* (*E-site*) is the location where the tRNA is discharged from the ribosome.

## Initation
The initiator codon is *AUG* which adds the *met* amino acid. The anitcondon is located on the *initiator tRNA*. The *mRNA binding site* is located on the small subunit. After, the large subunit is added to form the *translation iniation complex*.

The small sub-unit recognizes the 5' cap. It then scans down the mRNA until it reaches the AUG codon.

## Cycle
The first part is the cycle is codon recognition when the tRNA attaches to the A-site (GTP to GDP). The next event that occurs is the formation of the peptide bond and the attachment of the polypeptide attachment to the A-site. The two tRNAs then translocate to the next positions (GTP to GDP). The tRNA in the E-site then exits the ribosome.

## Termincation
The *release factor* bonds with one of the three *stop codons*. This structure also dissociates the structure and breaks the bond between the tRNA and polypeptide.

## Post-Translation
The folding structure is determined by primary structure. Sometimes, there are cleavages of polypeptides and the *C-peptide* is removed. For example, insulin has part of the peptide removed and the two new parts bind together. Furthermore, there is additions of sugars, phosphates and other molecules (*phosphorylation*, *glycosylation*). This may occur in the *lumin* of the rough cytoplasmic reticulum.

For example, *kinase* is an ezyme which phosphorolates proteins to add a phosphate group (*phosphorylation*).

## Genetic Code & Mutations
This code is redundant and non ambiguous. Each dna triplet codes for one specific item.

Changes in this genetic code, *mutations*, change the sequence of proteins and RNAs. These changes are passed on the offspring. *Mutagens* can be physical or chemical and can cause these mutations, along with errors in the replication process. *Frameshifts* may also occur due to insertions or deletions.

Many mutations can be detrimental. Some are "silent". Some can even be advantageous. One example is the *β-globin* protein which aggregates and forms into sickle shaped proteins.
